go.mod

```diff
@@ -1,12 +1,20 @@
 module github.com/ONSdigital/dp-topic-api
 
-go 1.13
+go 1.15
 
 require (
 	github.com/ONSdigital/dp-healthcheck v1.0.5
+	github.com/ONSdigital/dp-mongodb v1.4.0
```

Comment:
These dependencies are not used, can you run `go mod tidy`? then the unused dependencies will be removed from go.mod and go.sum

Reply:
done.